app/validators/proxy_features_validator.rb

```diff
@@ -6,7 +6,7 @@ def initialize(args)
 
   def validate_each(record, attribute, value)
     if value && !value.has_feature?(@options[:feature])
-      record.errors["#{attribute}_id"] << _(@options[:message])
+      record.errors["#{attribute}_id"] << _('does not have the %s feature') % @options[:feature]
```

Comment:
Best to prefer `@options[:message]` here if it's set, else (`||`) use the built in one - it's more like Rails ones then, and it's possible a plugin is using this validator.